awscrt/mqtt.py

```diff
@@ -435,6 +440,12 @@ def on_disconnect():
 
         try:
             _awscrt.mqtt_client_connection_disconnect(self._binding, on_disconnect)
+
+            if self.using_static_defaults:
+                ClientBootstrap.release_static_default()
+                DefaultHostResolver.release_static_default()
+                EventLoopGroup.release_static_default()
+
```

Comment:
Correct me if I missed anything, I think it might be better to have Client to handle the release like what we have in crt-java, since the bootstrap is referenced in Client class instead of the Connection.

Reply:
Good call! I'll adjust it 👍